### ISSUE TITLE
[dwell.com] Compatibility Report

### ISSUE THREAD

**Issue by MattAKAFred** [AdAway/AdAway#2878]
**Describe the third party app**

- Name: Dwell.com
- Home design magazine/website

**Describe the compatibility issue**
All assets/images are blocked by AdAway hosts file

**Workaround**
If applicable, describe the whitelist domains to add to fix the issue:

- assets.dwell.com
- images.dwell.com

(pulled the template from another issue, hope everything is ducky)
